v0/destinations/bq/transform.js

```diff
@@ -1,11 +1,12 @@
 const { processWarehouseMessage } = require("../util");
 
-function processSingleMessage(message, destination) {
-  return processWarehouseMessage("bq", message);
+function processSingleMessage(message, destination, schemaVersion) {
+  return processWarehouseMessage("bq", message, schemaVersion);
 }
 
 function process(event) {
-  return processSingleMessage(event.message, event.destination);
+  const schemaVersion = event.request.query.whSchemaVersion;
+  return processSingleMessage(event.message, event.destination, schemaVersion);
```

Comment:
We don't need event.destination as the 2nd argument.

Reply:
removed